scripts/performance/perf_load/perf_processes.py

```diff
@@ -95,13 +95,20 @@
 parser.add_argument('--test_conn', action='store_true', dest='test_conn',
                     help='Check pool connection with provided genesis file')
 
+parser.add_argument('--taa_text', default="", type=str, required=False,
```

Comment:
Maybe it would be more convenient for our use cases to have a default TAA?

Reply:
I'm inclined to agree